src/pybind/mgr/dashboard/frontend/src/styles.scss

```diff
@@ -47,6 +47,11 @@ $form-feedback-font-size: 100%;
   }
 }
 
+// TODO: move to somewhere else later
+.no-margin-bottom .alert {
+  margin-bottom: 0;
+}
+
```

Comment:
```suggestion
```
This can be removed.

Reply:
Seems this can't be removed.
You can move this to the component scss and, if needed, use `::ng-deep` to apply the style.